atlasdb-console/src/main/groovy/com/palantir/atlasdb/console/module/AtlasCoreModule.groovy

```diff
@@ -141,7 +143,7 @@ class AtlasCoreModule implements AtlasConsoleModule {
     }
 
     private setupConnection(AtlasDbConfig config) {
-        SerializableTransactionManager tm = TransactionManagers.create(config, ImmutableSet.<Schema>of(),
+        SerializableTransactionManager tm = TransactionManagers.create(config, Optional.<SSLSocketFactory>absent(), ImmutableSet.<Schema>of(),
```

Comment:
On second thought I don't want to to do that here, because create has the proper logic around understanding if we're embedded, trying to connect to an external set of timelock, or a leader ourselves (the latter of which is impossible here given we don't let console run with leader config, but you get the idea), which affects whether or not the ssl stuff exists and where it'll be.

Which is an unfortunate piece to this.  The atlas config has so much repetition that the sslConfiguration then suffers from it as well.